### PR TITLE
[REF] mail: `MessageInReplyToView`, review and simplify scss

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2826587

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hello @seb-odoo @alexkuhn 👋
This PR is ready for review 👍

cc. @stefanorigano

File: addons/mail/static/src/components/message_in_reply_to_view/message_in_reply_to_view.xml [hunk 1]
```diff
@@ -2,7 +2,8 @@
 <templates xml:space="preserve">
     <t t-name="mail.MessageInReplyToView" owl="1">
         <t t-if="messageInReplyToView">
-            <small class="o_MessageInReplyToView text-small pt-1 pr-2 position-relative" t-attf-class="{{ className }}" t-ref="root">
+            <small class="o_MessageInReplyToView position-relative d-block pt-1 pr-2 text-small text-truncate o_cursor_pointer" t-attf-class="{{ className }}" t-ref="root">
+                <span class="o_MessageInReplyToView_corner position-absolute bottom-0 d-block mb-1 pr-4 pb-2 opacity-50"/>
```
Comment (alexkuhn) [inline addons/mail/static/src/components/message_in_reply_to_view/message_in_reply_to_view.xml:6]: Style was `d-inline-block`, which is the case by default on `span`.
From my testing, `d-block` is actually useless: visual is the same with or without it.

**Review by alexkuhn — CHANGES_REQUESTED** [review]

**Comment by Brieuc-brd** [comment]
@alexkuhn The branch is updated 👍

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**